packages/amplication-server/src/core/resource/resourceBtm.service.ts

```diff
@@ -276,12 +319,36 @@ export class ResourceBtmService {
     );
   }
 
+  findUniqueEntities(
```

Comment:
@abrl91 why not use Set.difference() function which is meant for this exactly (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set/difference) ?